DOCUMENTATION.md

```diff
@@ -312,6 +319,13 @@ By default, you can not overwrite a package in the target team. To do this you c
 content-cli import packages -p <profileName> --file <exportedPackagesFile> --dataModelMappingsFile <dataModelMappingsFile> --overwrite
 ```
 
+If you want to not overwrite a package but not the Action Flows inside that package you can use the --excludeActionFlows flag
```

Comment:
nit: If you **want**

CDP service would have to send this new flag always, to fix the bug. Were they notified of the change?

Reply:
I contacted with CDP team. After we merge this change. They will be able to upgrade the version of content-cli and adapt the changes on their end.